Makefile

```diff
@@ -1,8 +1,8 @@
 NAME          := hdf5-vol-daos
 SRC_EXT       := gz
 TEST_PACKAGES := daos-devel $(NAME) $(NAME)-mpich $(NAME)-openmpi3 $(NAME)-mpich-devel $(NAME)-openmpi3-devel $(NAME)-mpich-tests $(NAME)-openmpi3-tests
-SOURCE_COMMIT := fcbdc0b32de8792d029506af4b45c490b9f9aacc
-TEST_COMMIT   := d3f80a57ca252eaddd1978496a0cd7a56a7fdc09
+SOURCE_COMMIT := 9eebd906b4b3b40aceac62da6c3d896fdb3a1d6e
```

Comment:
I thought we let this kind of build slide the last time because there was a commitment to bring it up to (more) consistent standards (a release plus a patch if some random, non-tagged/released build is necessary).

This is particularly easy now, given that 9eebd906b4b3b40aceac62da6c3d896fdb3a1d6e is actually an upstream tag.

There is no reason that this repo should not look and operate a lot more like the many other repos that build on tags/releases, without all of the added hackery such as the `curl` recipes in this file.